examples/client-demo/readme.md

```diff
@@ -36,8 +36,31 @@ $ export API_KEY=<your-api-key>
 $ export API_SECRET=<your-api-secret>
 ```
 
-After these have been set, you can run the application using the following command.
+### Basic Client Test
+If you simply want to test the compatibility of the client in regards to your platform, use the following
+command:
 
 ```
-$ java -jar client-demo-1.0-SNAPSHOT-all.jar.jar
+$ java -jar client-demo-1.0-SNAPSHOT-all.jar
 ```
+
+This will issue a few test requests to verify it working on your platform.
```

Comment:
... to verify [the System, the Client, etc.] working properly on your platform. 

Might sound better.